src/main/docs/guide/quickStart.adoc

```diff
@@ -156,9 +156,14 @@ public class MyResourceGenerator implements AOTCodeGenerator {
 
 Then in a configuration file, the code generator would be configured this way:
 
-```
```

Comment:
:x: This must be reverted: this is not a Micronaut configuration file, but the AOT configuration file which is independent and uses only properties files.